samza-test/src/test/resources/test-config.prop

```diff
@@ -0,0 +1,26 @@
+#
+#
```

Comment:
Is this file used?

Reply:
Missed this in the merge. This is no longer used. Deleted.